### PR TITLE
cbmp: init at 1.1.1

### PR DESCRIPTION
**Pull request by mrtnvgr**
## Description of changes

Closes #261224 

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/cb/cbmp/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  env = {
+    PUPPETEER_SKIP_DOWNLOAD = true;
+  };
+
+  postPatch = ''
+    cp ${./package-lock.json} package-lock.json
+  '';
+
+  meta = with lib; {
```
Comment (yunfachi) [inline pkgs/by-name/cb/cbmp/default.nix:29]:
```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/292468

File: pkgs/by-name/cb/cbmp/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    description = "CLI App for converting cursor svg file to png";
+    homepage = "https://github.com/ful1e5/cbmp";
+    license = licenses.mit;
```
Comment (yunfachi) [inline pkgs/by-name/cb/cbmp/default.nix:32]:
```suggestion
    license = lib.licenses.mit;
```

**Review by yunfachi — COMMENTED** [review]
you should also add yourself to the maintainers of this package
https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md

File: pkgs/by-name/cb/cbmp/package.nix [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  buildNpmPackage,
+  fetchFromGitHub,
+  lib,
+  ...
+}:
```
Comment (SuperSandro2000) [inline pkgs/by-name/cb/cbmp/package.nix:6]:
```suggestion
}:

```

File: pkgs/by-name/cb/cbmp/package.nix [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
+  buildNpmPackage,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildNpmPackage rec {
```
Comment (FliegendeWurst) [inline pkgs/by-name/cb/cbmp/package.nix:7]: Can you try using `yarn{Config,Build}Hook` instead of buildNpmPackage?

Comment (SuperSandro2000) [reply pkgs/by-name/cb/cbmp/package.nix:7]: I don't think this applies here.

Comment (FliegendeWurst) [reply pkgs/by-name/cb/cbmp/package.nix:7]: Why not? There is a `yarn.lock` in the repository.

**Pull request merged**